Otherworldly Pines/Assets/Scripts/Gravity Flip/GravityControl.cs

```diff
@@ -1,18 +1,37 @@
-﻿using System.Collections;
+﻿using System;
+using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 
 public class GravityControl : GravityAffected, IHUDConnected {
 
     private GravityFlipIndicator indicator;
     private GravityRegion activeGravityRegion;
+    private PlayerMovement player;
```

Comment:
Oh also, this has been renamed to PlayerControls

Reply:
Fixed